main.go

```diff
@@ -95,7 +95,7 @@ func generateFile(gen *protogen.Plugin, file *protogen.File) {
 			content, _ := g.Content()
 
 			g2.P("package " + file.GoPackageName)
-			body := fmt.Sprintf("const %s=`%s`", tableName, content)
+			body := fmt.Sprintf("const %s_parquet_schema=`%s`", strings.Title(tableName), content)
```

Comment:
To be honest i prefer to have something like 
this will be more Go'ish: 
```go
fmt.Sprintf(ParquetSchema%s`, camelCase(tableName))
...
func camelCase(s string) string {
	p := strings.Fields(s)
       
        res := []string{}
	for _,value := range p {	
		g = append(g,strings.Title(value))
	}	
	
	resturn strings.Join(res,"")
}
```
WDYT?

Reply:
Going with snake case all upper case for constants.